_includes/functions/details-list.md

```diff
@@ -12,13 +12,15 @@
 {% for i in (0..9) %}
   {% capture q %}q{{i}}{% endcapture %}
   {% capture a %}a{{i}}{% endcapture %}
+  {% capture al %}a{{i}}link{% endcapture %}
+  {% capture alink %}{% if include.[al] != nil %}<a href="{{ include.[al] }}" class="external"> </a>{% endif %}{% endcapture %}
   {% if include.[q] == nil %}{% break %}{% endif %}
   {% assign qslug = include.[q] | slugify: 'latin' %}
   {% if jekyll.environment == "email" %}
-   - <i markdown="1">{{include.[q]}}</i><br>{{include.[a]}}
+   - <i markdown="1">{{include.[q]}}</i><br>{{include.[a]}} {{alink}}
```

Comment:
I think a `&nbsp;` would be better than a regular space so that the external link icon doesn't appear on a line by itself.